velox/substrait/VeloxToSubstraitPlan.cpp

```diff
@@ -308,4 +318,94 @@ void VeloxToSubstraitPlanConvertor::toSubstrait(
   aggregateRel->mutable_common()->mutable_direct();
 }
 
+void VeloxToSubstraitPlanConvertor::toSubstrait(
+    google::protobuf::Arena& arena,
+    const std::shared_ptr<const core::OrderByNode>& orderByNode,
+    ::substrait::SortRel* sortRel) {
+  const auto& source = getSingleSource(orderByNode);
+  toSubstrait(arena, source, sortRel->mutable_input());
+
+  sortRel->MergeFrom(processSortFields(
+      arena,
+      orderByNode->sortingKeys(),
+      orderByNode->sortingOrders(),
+      source->outputType()));
+
+  // TODO: Enable this after substrait add this field.
+  // sortRel->set_is_partial(orderByNode->isPartial());
+  sortRel->mutable_common()->mutable_direct();
```

Comment:
What does this code do?

Reply:
like the output type in velox, this is the output for the substrait rel.

Reply:
@ZJie1 This code is confusing to me because it seems like it doesn't do anything, does it? We are capturing the return value and not using it. What happens if we remove this code?